src/app/nav-bar/nav-bar.component.ts

```diff
@@ -1,9 +1,17 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'app-nav-bar',
   templateUrl: './nav-bar.component.html',
   styleUrls: ['./nav-bar.component.css']
 })
-export class NavBarComponent {
+export class NavBarComponent implements OnInit {
+  constructor(private router: Router) {}
+
+  ngOnInit(): void {
+    this.router.routeReuseStrategy.shouldReuseRoute = () => false;
```

Comment:
What's this for? Does it force angular to re-render components that have similar routes? I can't remember. But this nav-bar reminds me of the browser tabs that Shure implemented. People did not want to force a reload since each screen has so much UI stuff in it...even though it makes the code more complicated.

Reply:
Yeah, I added this because I noticed that previously if you completed a search, clicking the Search link in the navbar wouldn't refresh the page or have any effect. I looked into this and read that Angular won't reload a component on the same route unless you explicitly tell it to by adding this code

It makes sense that forcing a reload would be undesirable on a page with components you want to keep, but this actually doesn't seem to have that effect on the landing page?

Reply:
This is really tricky. I'm going to approve, but you should be aware that the router is a singleton and that overriding `shouldReuseRoute` here it overrides that function for the router throughout the app. So it becomes a kind of hidden override. Maybe we add a card to the backlog to address this?

So far I'm seeing this link https://stackoverflow.com/questions/49155895/how-to-activate-routereusestrategy-only-for-specific-routes which would let you make it obvious that this app uses your CustomRouteReuseStrategy...
Or you could listen for router events inside the component and zero out the variables whenever a NavigationEnd happens.

Reply:
Ah cool, I can definitely create a `CustomRouteReuseStrategy`... would you rather I implement that in this ticket or create a new ticket for it?

Reply:
Create a new one. I think the other things are more important.